bokehjs/src/coffee/models/tools/gestures/pan_tool.coffee

```diff
@@ -39,44 +39,44 @@ class PanToolView extends GestureTool.View
     new_dx = dx - @last_dx
     new_dy = dy - @last_dy
 
-    hr = _.clone(frame.get('h_range'))
```

Comment:
@bryevdv, was this ever necessary to clone this? Currently it's not possible to clone using `_.clone()`. I'm not sure if it was ever working as expected. Currently the safest is to use:
```
frame.h_range.constructor(frame.h_range.attributes)
```